Library/Formula/czmq.rb

```diff
@@ -43,13 +42,9 @@ def install
 
     system "./autogen.sh" if build.head?
     system "./configure", *args
-    system "make", "install"
+    system "make"
+    system "make check"
```

Comment:
Okay, I was following the pattern of other Formulae. I won't comment on whether it looks nicer or not.